frontend/src/components/regions/CardDetails/Review.js

```diff
@@ -46,9 +60,7 @@ const useStyles = makeStyles((theme) => ({
 
 const Review = ({ review }) => {
   const classes = useStyles();
-
-  // const [expand, setExpand] = useState(false);
-  // const onClick = () => setExpand(!expand);
+  const [openReviewPopUp, setOpenReviewPopUp] = useState(false);
```

Comment:
This kind of logic should not be inside this file. This is a presentation component. 
Some components are about logic, and others are for display.  If we are consistent then everyone will know what to expect.